docs/installation.md

```diff
@@ -73,7 +73,7 @@ docker-compose.yaml for deployment. The example files required are in
 docker-compose up
 ```
 
-Then visit via your web browser http://localhost:5000
+Then visit via your web browser http://localhost:5000. The default username is `admin` and password is `test`.
```

Comment:
Any username works the password though however has to be `test`. This is since we are using https://github.com/Quansight/conda-store/blob/main/tests/assets/jupyterhub_config.py#L4

Reply:
Thanks for clarifying! I updated the text.